.buildkite/bk.integration.pipeline.yml

```diff
@@ -61,6 +61,41 @@ steps:
         matrix:
           - default
 
+      - label: "Win2025:sudo:{{matrix}}"
```

Comment:
IIUC, the same steps apply for `platform-ingest-elastic-agent-windows-2022`, so reusing the step with a new axis in the matrix might help maintenance. Why is there no matrix with the image name?